test/steps/browser.js

```diff
@@ -75,6 +94,16 @@ async function thenISee(expected: string): Promise<void> {
     await assert.textIsVisible(this.driver, expected);
 }
 
+async function thenIDontSee(expected: string): Promise<void> {
+    try {
+        await assert.textIsVisible(this.driver, expected);
```

Comment:
Is there some kind of timeout here? Will this give a false negative if the content is loading asynchronously? (I don't know much about promises so these not be relevant questions.)